AzureFunctions.AngularClient/src/app/extension-install/extension-install.component.scss

```diff
@@ -0,0 +1,18 @@
+.wrapper {
+    position: relative;
+}
+
+.highlight {
+    padding: 10px 5px 10px 5px;
+    padding-bottom: 10px;
+    background-color: #E6E7E8;
```

Comment:
Move color to _variable.cscss or use:
$background-color-opaque: #ccc;
$background-color-disabled: rgba(127, 127, 127, 0.5);
$grey-color: #f5f5f5;